### ISSUE TITLE
BUG: SQLite - Cannot read properties of null (reading 'byteLength')

### ISSUE THREAD

**Issue by molarmanful** [beekeeper-studio/beekeeper-studio#2675]
**Describe the bug**
Similar to #2640, SQLite tables with nullable blob columns error with `Cannot read properties of null (reading 'byteLength')` when a row with a null blob exists.

**To Reproduce**
1. Create a table with a nullable column of type blob (`column BLOB NULL`)
2. Add a row but leave the blob column as `(NULL)`.

**Expected behavior**
The table should be able to display the row with the null blob without error.

**Version Information (please complete the following information):**
 - OS: Windows 11
 - App Version v5.0.9
 - Database type and version: SQLite v3.47.2

**Comment by rathboma** [comment]
@azmy60 looks like a binary issue?

**Comment by azmy60** [comment]
Yeah. Definitely need a better way to serialize the binary columns.

**Comment by aarbmx6s** [comment]
The same issue.
- OS: macOS 15
- Database type and version: PostgreSQL v17
- Column type: `bytea`.
